finality-provider/service/fp_instance.go

```diff
@@ -275,6 +219,16 @@ func (fp *FinalityProviderInstance) finalitySigSubmissionLoop() {
 					zap.Uint64("last_processed_height", res.LastProcessedHeight),
 				)
 
+				// start poller after fast sync is finished
+				if !fp.poller.IsRunning() {
+					err := fp.poller.Start(res.LastProcessedHeight + 1)
+					if err != nil {
+						fp.logger.Error("failed to start the poller", zap.Error(err))
+						fp.reportCriticalErr(err)
+					}
+					continue
+				}
```

Comment:
I don't think the start of the poller should be controlled within the sig submission loop. The poller should be already started when the loop is on